packages/utils/style/layout.ts

```diff
@@ -3,7 +3,7 @@ export const headerHeight = 80;
 export const MOBILE_HEADER_HEIGHT = 64;
 export const legalFooterHeight = 58;
 export const screenContentMaxWidth = 1092;
-export const screenContentMaxWidthLarge = 1600; //"100%";
+export const screenContentMaxWidthLarge = 1290; //"100%";
```

Comment:
This change is significant. Can you please provide more information about why it is necessary?

Reply:
We quickly discuss about this here : https://discord.com/channels/876922049279455274/993144491181617162/1145698425648713768

1600px doesn't make sense. Regarding to the Figma, our max width ix 1290. Or we can set `fullWidth` on `ScreenContainer` (I'm not sure `fullWidth` is still working)